### PR TITLE
HTML5 Form Validation

### PR DESCRIPTION
**Pull request by Phhere**

### CODE REVIEW

**Comment by evgeni** [comment]
Can I have English commit messages and one commit only? :)

File: index.html [hunk 1]
```diff
@@ -86,7 +86,9 @@ <h1 id="showssid"></h1>
         };
         
         $(document).ready(function(){
-            $('#generate').click(function() {generate()});
+            $('#form').submit(function() {
+                 generate();
+                return false; });
```
Comment (evgeni) [inline index.html]: there is a line break missing in action here :)

File: index.html [hunk 2]
```diff
@@ -86,7 +86,9 @@ <h1 id="showssid"></h1>
         };
         
         $(document).ready(function(){
-            $('#generate').click(function() {generate()});
+            $('#form').submit(function() {
```
Comment (evgeni) [inline index.html:89]: i wonder why this works. the button above is a button, not a submit button...

Comment (Phhere) [inline index.html:89]: http://www.w3.org/html/wg/drafts/html/master/forms.html#the-button-element
"The missing value default is the Submit Button state."

It's the normal behaviour ;)

Comment (evgeni) [inline index.html:89]: interesting. my memory told me buttons are only submit buttons when type=submit or they are defined directly via <submit>, but fine for me :)

**Comment by Phhere** [comment]
Merged into one commit with english description ;)

**Pull request merged**